pghoard/transfer.py

```diff
@@ -174,19 +175,14 @@ def increment(self, file_key: str, total_bytes_uploaded: float) -> None:
                             file_key,
                             stalled_age,
                         )
-                metric_data = {
-                    "metric": "pghoard.basebackup_bytes_uploaded",
-                    "inc_value": total_bytes_uploaded,
-                    "tags": {
-                        "delta": file_type in (FileType.Basebackup_delta, FileType.Basebackup_delta_chunk)
-                    },
-                }
-            elif file_type in (FileType.Wal, FileType.Timeline):
-                metric_data = {"metric": "pghoard.compressed_file_upload", "inc_value": total_bytes_uploaded}
+                upload_percentage = (total_bytes_uploaded / file_size) * 100
```

Comment:
Can file_size be zero?

Reply:
There won't be a file_size with zero; The increment call is based on the response from the rohmu callback, and I believe there is a limit, possibly around 5mb

Reply:
I'd just handle the zero case here anyway, can't hurt to be sure we don't get a divide by zero error if something changes later in how things are working.

Reply:
added a if condition